test/addons/callback-scope/test-resolve-async.js

```diff
@@ -4,6 +4,8 @@ const common = require('../../common');
 const assert = require('assert');
 const { testResolveAsync } = require(`./build/${common.buildType}/binding`);
 
+common.crashOnUnhandledRejection();
```

Comment:
I'm with this but there is an assertion below in the `setTimeout()` callback which should make this redundant.

Reply:
Should I make any changes?

Reply:
@lpinca `common.crashOnUnhandledRejection();`  would display the rejection in the trace instead of a timeout though.

Reply:
Yes that's better. :shipit: